src/tribler-core/tribler_core/modules/metadata_store/restapi/metadata_endpoint.py

```diff
@@ -12,6 +12,9 @@
 from tribler_core.utilities.unicode import hexlify
 
 
+default_timeout = 20
```

Comment:
Please, make this ALL_CAPITAL_LETTERS. Module-level constants should be all caps.

Reply:
Also, the variable name should reflect *what kind* of timeout that is. Like, `TORRENT_CHECK_TIMEOUT`.
These measures are necessary to prevent name clashes and enhance readability.

Reply:
done.